.eslintrc.json

```diff
@@ -11,292 +19,20 @@
         "libraries/streaming-e2e-tests/**"
     ],
     "parser": "@typescript-eslint/parser",
-    "plugins": [
-        "@typescript-eslint",
-        "only-warn"
-    ],
-    "overrides": [
-        {
-            "files": ["libraries/adaptive-expressions/src/*.ts", "libraries/adaptive-expressions/src/**/*.ts"],
```

Comment:
@stevengum where did we land on dropping jsdoc linting? I thought it might still be useful to include as a warning but I could be mistaken there.

Reply:
@JonathanFingold can you indicate what the correct practice is here? More specifically which tools and configurations (if any) should we leave in the JS repo for generating the docs?

Reply:
Researching. I'll post here when I know more.

Reply:
My plan is to restore the js-doc - maybe with fewer rule exceptions - once we have the code issues fixed.

Reply:
### TypeDoc

We use TypeDoc behind the scenes for TypeScript documentation (recommended reading: [platform breakdown](https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/concepts/platforms)). To make sure that the content renders properly, please ensure that the TypeScript APIs are using standard TypeDoc conventions, available [here](http://typedoc.org/guides/doccomments/).

### Linting & structure enforcement

You can use [ESLint](https://eslint.org/), in combination with `valid-jsdoc` to make sure that your JSDoc comments are in good shape.

For TypeScript, you can use [tslint](https://github.com/palantir/tslint), with the `completed-docs` [rule](https://palantir.github.io/tslint/rules/completed-docs/).

---

Use `[link-text](xref:uid)` style links to link to other Bot Framework and Microsoft TypeScript ref docs.

To find the xref for JavaScript, go to https://docs.microsoft.com/en-us/javascript/api/ and search for the member you want to link to. The xref will be the link text, as opposed to the link target. For example, the xref for the `BotAdapter.use` method is **botbuilder-core.BotAdapter.use**.

For links into our docs, link "directly" to the article, such as https://docs.microsoft.com/azure/bot-service/bot-builder-howto-send-messages for the **Send and receive text messages** how to.